src/instances/HeadcountInstance.ts

```diff
@@ -577,15 +590,23 @@ export class HeadcountInstance {
     /**
      * Converts a headcount.
      * @param {GuildMember} member The person that converted this.
-     * @param {VoiceChannel | null} vcToUse The voice channel to use, if any.
+     * @param {VoiceChannel | boolean} vcToUse The voice channel to use, or true if vcless, false otherwise.
```

Comment:
You got it.

This method was used to pick a vc, and if it returned null that meant create a temporary vc.  Since we now have three cases (existing vc, vcless, or temporary vc), I needed to find a way to return these three options. This was the most convenient.